### PR TITLE
Update package metadata

### LINKED ISSUE #68: Package metadata
The current package metadata in [`setup.cfg`](https://github.com/czbiohub/iohub/blob/efbb7f4c9fb992ca332b3d403404efbe22aa0002/setup.cfg#L36) is inherited from waveorder and saw minor revisions. As iohub approaches a PyPI release, we need to agree on some update to the information. (See comments in the code blocks below)

Author and email:

```properties
author = Computational Microscopy Platform, CZ Biohub  # change to CZ Biohub?
author_email = shalin.mehta@czbiohub.org.  # does this need to change?
```

Classiffiers:

```properties
classifiers =
    Development Status :: 4 - Beta. # Alpha?
    Intended Audience :: Science/Research
    License :: OSI Approved :: BSD License  # is 3-clause the original BSD?
    Programming Language :: Python :: 3 :: Only
    Topic :: Scientific/Engineering
    Topic :: Scientific/Engineering :: Image Processing
    Topic :: Scientific/Engineering :: Visualization  # remove
    Topic :: Scientific/Engineering :: Information Analysis  # remove? this is too generic
    Topic :: Scientific/Engineering :: Bio-Informatics  # remove
    Operating System :: Microsoft :: Windows
    Operating System :: POSIX
    Operating System :: Unix
    Operating System :: MacOS
    # any additional classifiers? 
```

@mattersoflight @AhmetCanSolak @JoOkuma would love to have your input on this.

### PR DESCRIPTION
**Pull request by ziw-liu**
Resolves #68.

### CODE REVIEW

**Comment by ziw-liu** [comment]
Tagging more people on the reviewers list since this PR changes the `author` field.

File: setup.cfg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = iohub
-author = Computational Microscopy Platform, CZ Biohub
-author_email = shalin.mehta@czbiohub.org
+author = CZ Biohub and iohub contributors
+author_email = iohub@czbiohub.org
```
Comment (AhmetCanSolak) [inline setup.cfg:4]: is the email ready-to-use already @ziw-liu ? could you send a test email to make sure.

**Review by AhmetCanSolak — APPROVED** [review]
If desired we can have an authors.md file in future too.

**Comment by mattersoflight** [comment]
LGTM

**Pull request merged**